admin-node-setup.sh

```diff
@@ -48,3 +48,6 @@ cp $tmp_dir/public.yaml $kube_dir
 cp $tmp_dir/private.yaml $kube_dir
 
 rm -rf $tmp_dir
+
+# switch deprecated --config flag in kubelet
+sed -i -e "s/--config/--pod-manifest-path/g" /etc/kubernetes/kubelet
```

Comment:
Nit: I'd include `--config=` and `--pod-manifest-path=`. Just to make sure we don't replace anything starting with `--config`. It probably doesn't exist (don't have a cluster at hand to check), but just for sanity.

Reply:
done